src/mixins/java/org/spongepowered/common/mixin/api/minecraft/world/level/chunk/LevelChunkMixin_API.java

```diff
@@ -330,4 +358,96 @@ public boolean isAreaAvailable(final int x, final int y, final int z) {
         return VecHelper.inBounds(x, y, z, this.blockMin(), this.blockMax());
     }
 
+    @SuppressWarnings({"unchecked", "rawtypes"})
+    @Override
+    public Collection<? extends Player> players() {
+        return (Collection) SpongeCommon.server().getPlayerList().getPlayers().stream()
```

Comment:
Would there be less looping by asking the `ServerLevel` for the players if the `ServerLevel` actually is loaded?